vivisect/qt/ctxmenu.py

```diff
@@ -53,6 +53,27 @@ def printEmuState(vw, fva, va):
 
         vw.vprint(base)
 
+def newMemoryView(vw, xexpr):
+    try:
+        vw.getVivGui().newMemoryView(name="mem:%s" % xexpr, expr=xexpr)
+        
+    except:
+        logger.warning("Fail!", exc_info=1)
+
+def newFuncGraph(vw, xexpr):
+    try:
+        vw.getVivGui().newFuncGraphView(name="FG:%s" % xexpr, expr=xexpr)
+        
+    except:
+        logger.warning("Fail!", exc_info=1)
```

Comment:
Can we flesh out these log messages? Seeing just a generic `Fail!` in my terminal is not helpful. I'd like to at least see what viv was trying to do when this gets printed out.